src/Config.hs

```diff
@@ -41,8 +43,12 @@ data Config
     = Config
     { getPool :: ConnectionPool
     , getEnv  :: Environment
+    , getMetrics :: Metrics
     }
 
+instance Monad m => MonadMetrics (AppT m) where
+    getMetrics = liftM Config.getMetrics ask
```

Comment:
I generally prefer to see `fmap` instead of `liftM`, though that's not a big deal.

You can also use `asks :: MonadReader r m => (r -> a) -> m a`, as:

```haskell
getMetrics = asks Config.getMetrics
```

Reply:
Great stuff, thanks!